index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Richard's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Inter&family=Permanent+Marker&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+   <div class="app-wrap">
+     <header class="app-header">
+         <a >Welcome</a>
+
+         <nav><i class="fas fa-bars"></i></nav>
+     </header>
+     <!-- <div class="for_background"> -->
+
+       <section class="headline">
+         <div class="primary">
+            <h1>Hey there. I'm Richard</h1>
+
+            <p>I'm a software developer</p>
+         </div>
+          <div class="support">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+  don’t hestiate to contact me.</p>
+          </div>
+          <div class="social-media">
+            <a href="#"><i class="fab fa-twitter social"></i></a>
+            <a href="#"><i class="fab fa-linkedin social"></i></a>
+            <a href="#"><i class="fab fa-medium social"></i></a>
+            <a href="#"><i class="fab fa-github social"></i></a>
+            <a href="#"><i class="fab fa-angellist social"></i></a>
+
+          </div>
+
+          <div class="arrow">
+            <button  class="arrow-button" ><i class="fas fa-chevron-circle-down fa-3x"></i></button>
+          </div>
```

Comment:
- [ ] Kudos in using the correct chevron icon but your design differs from the given design. Please try to remove the black color from the chevron

Your Design | Figma Design  |
:---------------:|:--------------------:|
![d4](https://user-images.githubusercontent.com/39523312/148355568-28e8a389-b895-4585-89d2-4d84e982b12e.png) |![d3](https://user-images.githubusercontent.com/39523312/148355580-000da464-f718-4432-90a4-7caa97971eab.png)